apple/internal/apple_framework_import.bzl

```diff
@@ -616,8 +617,16 @@ def _common_static_framework_import_impl(ctx, is_xcframework):
     providers.append(
         _objc_provider_with_dependencies(ctx, objc_provider_fields, additional_objc_infos),
     )
+
+    includes = []
+    if is_xcframework and not is_framework and ctx.attr.includes:
```

Comment:
Why is there a `not is_framework` here?

Reply:
Note that this has changed slightly at HEAD

Reply:
It looks to be broken at HEAD? It doesn't look like there is a prefix added but I'm not exactly sure where those are supposed to be relative too: https://cs.github.com/bazelbuild/rules_apple/blob/76752fe84ff0dc5d793033015647c78efbf41309/apple/internal/apple_xcframework_import.bzl#L578

Reply:
do you have a simple case I can test with? definitely possible something regressed